samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -215,7 +271,17 @@ static void button_handler_cb(uint32_t pressed, uint32_t changed)
 				      &bt_mesh_sensor_motion_threshold,
 				      &presence_motion_threshold[motion_threshold_idx++]);
 		if (err) {
-			printk("Error setting motion threshold setting (%d)\n", err);
+			printk("Error setting motion threshold setting for Presence Detected "
+			       "sensor (%d)\n", err);
+		}
+		motion_threshold_idx = motion_threshold_idx % ARRAY_SIZE(presence_motion_threshold);
+
+		err = setting_set_int(&bt_mesh_sensor_motion_sensed,
```

Comment:
Since you have the same backing value for both servers on the server side, this is a no-op (since the same state is written by the other setting set above). See my server side comment on this.